crates/rattler_conda_types/src/match_spec/parse.rs

```diff
@@ -527,13 +528,9 @@ fn matchspec_parser(
         let channel_config = ChannelConfig::default_with_root_dir(
             std::env::current_dir().expect("Could not get current directory"),
         );
-        if let Some((channel, subdir)) = channel_str.rsplit_once('/') {
```

Comment:
I'm not sure this was correct, as a channel can also be "conda-forge/label/xyz"

This is now why the test fails.

Reply:
After discussing with @baszalmstra we're going to reimplement this with a list of "known_subdirs" which is just the `platforms`.